### ISSUE TITLE
6. scipy

### ISSUE THREAD

**Issue by rkdarst** [AaltoSciComp/python-for-scicomp#8]
Check the pad, but as of this writing:

Scipy (Know that it exists):
- What is scipy?
  - Wrapper around C and fortran libraries
- When you need it, you will search for it

Scipy (Know that it exists):
- What is scipy?
  - Wrapper around C and fortran libraries
- When you need it, you will search for it

**Comment by annefou** [comment]
Exercise: interpolation

**Comment by bast** [comment]
Yes we should add that exercise. That's a good and relevant exercise.
